agent/config/config.go

```diff
@@ -297,6 +297,18 @@ func environmentConfig() Config {
 		seelog.Warnf("Invalid format for \"ECS_NUM_IMAGES_DELETE_PER_CYCLE\", expected an integer. err %v", err)
 	}
 
+	instanceAttributesEnv := os.Getenv("ECS_INSTANCE_ATTRIBUTES")
+	attributeDecoder := json.NewDecoder(strings.NewReader(instanceAttributesEnv))
+	var instanceAttributes map[string]string
+
+	err = attributeDecoder.Decode(&instanceAttributes)
+	if err != io.EOF && err != nil {
+		seelog.Warnf("Invalid format for ECS_INSTANCE_ATTRIBUTES. Expected a json hash")
```

Comment:
Shouldn't this result in a fatal error as well as this is same as a 'malformed attribute' error?